src/models/multiBarChart.js

```diff
@@ -334,12 +334,14 @@ nv.models.multiBarChart = function() {
 
     multibar.dispatch.on('elementMouseover.tooltip', function(evt) {
         evt.value = chart.x()(evt.data);
-        evt['series'] = {
-            key: evt.data.key,
-            value: chart.y()(evt.data),
-            color: evt.color
-        };
-        tooltip.data(evt).hidden(false);
+        if (!chart.stacked() || (evt.data.size > 0 || evt.data.size == undefined)) {
```

Comment:
Well, you need to really do multibar.getY(evt.data) I think, you can't assume that everyone will use the y attribute name since they can override that using the getY function :)